docs/feat/ui/ux.md

```
@@ -0,0 +1,811 @@
+- calm
+- mobile-native
+- easy to scan
+```
+
+---
+
+# 12) UX rules for the frontend team
+
+Use these rules when building screens:
+
+### Rule 1
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)